aws/table_aws_macie2_classification_job.go

```diff
@@ -0,0 +1,233 @@
+			{
+				Name:        "bucket_definitions",
+				Description: "The namespace of the AWS service that provides the resource, or a custom-resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "client_token",
+				Description: "The token that was provided to ensure the idempotency of the request to create the job.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+			{
+				Name:        "last_run_time",
+				Description: "This value indicates when the most recent run started.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+			{
+				Name:        "sampling_percentage",
+				Description: "The sampling depth, as a percentage, that determines the percentage of eligible objects that the job analyzes.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+			{
+				Name:        "custom_data_identifier_ids",
+				Description: "The custom data identifiers that the job uses to analyze data.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+			{
+				Name:        "last_run_error_status",
+				Description: "Specifies whether any account- or bucket-level access errors occurred when a classification job ran.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "s3_job_definition",
+				Description: "Specifies which S3 buckets contain the objects that a classification job analyzes, and the scope of that analysis.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+			{
+				Name:        "schedule_frequency",
+				Description: "Specifies the recurrence pattern for running a classification job.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+			{
+				Name:        "statistics",
+				Description: "Provides processing statistics for a classification job.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsMacie2ClassificationJob,
+			},
+
+			{
+				Name:        "user_paused_details",
+				Description: "Provides information about when a classification job was paused.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "The list of tags for the classification job.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsMacie2ClassificationJob,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsMacie2ClassificationJob,
+				Transform:   transform.FromField("Tags"),
```

Comment:
```suggestion
```

Reply:
What is datatype for `tags` in response?
Is it `key-value` object? 
If yes, can we remove `tags_src` column?